standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MsckInfo.java

```diff
@@ -33,6 +36,8 @@ public class MsckInfo {
   private final boolean dropPartitions;
   private final long partitionExpirySeconds;
 
+  private Map<String, String> smallFilesStats = new HashMap<>();
```

Comment:
it can be costly in terms of memory. isn't simple counter enough here?

Reply:
We need to show the end user that in this specific partition, there is a total size, and there is a total number of files, which means there are many small files existing in this partition.
And that can help the end user when they do the "msck" operation.